new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+
+  1. Move the cursor to the beginning of the first "xxx".
+
+  2. Now press «R» ([capital R](R)) and type the number below it in the second line, so that it
+     replaces the "xxx".
+
+  3. Write over the existing characters. If you make a mistake press backspace
+     to restore the character.
+
+  3. Press <Esc> to leave [Replace mode](mode-replace).  Notice that the rest of the line
+     remains unmodified.
+
+  4. Repeat the steps to replace the remaining xxx.
+
+---> Adding 123 to xxx gives you xxx. {expect:Adding 123 to 456 gives you 579.}
+---> Adding 123 to 456 gives you 579. {expect:Adding 123 to 456 gives you 579.}
+
+
+# Lesson something.something: BEGINNING AND END OF LINE MOTIONS
+
+Often times we want to do something with the beginning or the end of a line.
+There are two particular motions for this: «0» and «$».
+
+  1. Move your cursor on the first letter of the word "This".
+
+  2. Press «d0» to delete everything until the beginning of the line.
+
+  3. Press «$» to jump to the end of the line and press «i» to insert the
+     missing word.
+
+---> asdf This is a. {expect:This is a sentence.}
+---> This is a sentence. {expect:This is a sentence.}
+
+These two motions can be combined with any of the previous operators to affect
+everything on the line before or after the cursort. You can also use them to
+jump the cursor quickly around.
+
+We have already met the «D» and «C» operators which act exacly the same way as
+«d$» and «c$», so which one should you use? They are both equally repeatable
+because both are only once action, but the former require fewer keystrokes.
+
+
+# Lesson something.something: CHARACTER- AND LINEWISE MOTION
+
+** The «l» motion operates on a single character and «_» on the line **
+
+  1. Position your cursor on the p in "mption".
+
+  2. Press «cl» to enter insert mode, make the correction and exit back.
+
+---> Every mption we learn allows construction of new actions. {expect:Every motion we learn allows construction of new actions.}
+---> Every motion we learn allows construction of new actions. {expect:Every motion we learn allows construction of new actions.}
+
+Since this is a very common motion you can also use the space bar, it has the
+same effect. Pick whichever you prefer.
+
+Now let's try the line-wise motion.
+
+  1. Position your cursor anywhere on the line below.
+
+  2. Press «d_» to delete the entire line.
+
+---> This line has to go. {expect:NULL}
+
+Line-wise actions are very common and so there is a special convention: use the
+operator as motion as well to affect the entire line.
+
+  1. Press «cc» To change the contents of the entire line.
+
+---> Change this line. {expect:This is the way it should be.}
+---> This is the way it should be. {expect:This is the way it should be.}
+
+This is generally preferrable to the underscore motion above. Ultimately it
+makes no difference, but most instruction use the doubled form.
+
+
+# Lesson something.something: FINDING MOTION
```

Comment:
Maybe just `find` motion? I'm not sure if `finding` is how it's formally referred to.